mcs/class/corlib/Test/System/TimeZoneInfoTest.cs

```diff
@@ -359,6 +359,16 @@ public void TestAthensDST ()
 				Assert.IsFalse (tzi.IsDaylightSavingTime (date));
 				Assert.AreEqual (new TimeSpan (2,0,0), tzi.GetUtcOffset (date));
 			}
+
+			[Test] //Covers #41349
+			public void TestIsDST_DateTimeOffset ()
+			{
+				TimeZoneInfo tzi = TimeZoneInfo.FindSystemTimeZoneById ("Europe/Athens");
+				var date = new DateTime (2014, 3, 30 , 2, 0, 0);
+				var offset = tzi.GetUtcOffset (date);
+				var dateOffset = new DateTimeOffset (date, offset);
+				Assert.IsFalse (tzi.IsDaylightSavingTime (dateOffset));
```

Comment:
Can you please add another case that tests a time where IsDaylightSavingTime == true?